lib/resolveurl/plugins/waaw.py

```diff
@@ -22,146 +24,27 @@ class WaawResolver(ResolveUrl):
     name = "waaw"
     domains = ["waaw.tv", "hqq.watch", "netu.tv", "hqq.tv", "waaw1.tv"]
     pattern = "(?://|\.)((?:waaw1?|netu|hqq)\.(?:tv|watch))/(?:watch_video\.php\?v|.+?vid)=([a-zA-Z0-9]+)"
-
+    
     def __init__(self):
         self.net = common.Net()
-
+    
     def get_media_url(self, host, media_id):
         web_url = self.get_url(host, media_id)
         headers = {'User-Agent': common.FF_USER_AGENT,
-                   'Referer': 'https://waaw.tv/watch_video.php?v=%s&post=1' % media_id}
```

Comment:
may still need `'Referer': web_url` in headers, please check

Reply:
Playing is correct without referer.

Reply:
okay in which case why is web_url even defined, if it is not going to be used